app/assets/stylesheets/views/_consultation.scss

```diff
@@ -55,3 +15,7 @@
     }
   }
 }
+
+.consultation-date {
```

Comment:
As this isn't a BEM class, it should probably still live within the `.consultation` scope.